src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return String.valueOf(Color.values()[new Random().nextInt(Color.values().length)]);
```

Comment:
You can use static variable for Color.values().length to avoid calling length each time when getRandomColor() method executing.
Also, you can create inner variable 'int index' for better readability:
 int index = new Random().nextInt(arrayLength);